plugins/tableselection/plugin.js

```diff
@@ -789,6 +788,11 @@
 			return;
 		}
 
+		// #(547)
+		if ( evt.data.method === 'drop' ) {
+			return;
+		}
```

Comment:
I'm assuming this is needed to prevent this listener from executing whenever the 'paste' is performed because of the `drop` operation?

Reply:
Exactly. Without preventing d&d images breaks because of the logic in this function.